test/common/stats/stats_matcher_impl_speed_test.cc

```diff
@@ -39,7 +39,7 @@ class StatsMatcherPerf {
 } // namespace Stats
 } // namespace Envoy
 
-static void BM_Inclusion(benchmark::State& state) {
+static void bmInclusion(benchmark::State& state) {
```

Comment:
To be consistent with the other benchmarks you can leave this BM_ and use a NOLINT pragma.